src/main/java/uk/gov/hmcts/cmc/claimstore/services/document/AlwaysGenerateDocumentsService.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmcts.cmc.claimstore.services.document;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.documents.CountyCourtJudgmentPdfService;
+import uk.gov.hmcts.cmc.claimstore.documents.DefendantResponseCopyService;
+import uk.gov.hmcts.cmc.claimstore.documents.LegalSealedClaimPdfService;
+import uk.gov.hmcts.cmc.claimstore.documents.SettlementAgreementCopyService;
+import uk.gov.hmcts.cmc.claimstore.services.ClaimService;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+@Service
+@ConditionalOnProperty(prefix = "feature_toggles", name = "document_management", havingValue = "false")
+public class AlwaysGenerateDocumentsService implements DocumentsService {
```

Comment:
What do you think about `OnTheFlyDocumentsService`? (after I typed that in I realised it probably looks as awkward as `AlwaysGenerateDocumentsService` :p)

Reply:
how about DefaultDocumentsService??

Reply:
I kind of like the name as this is descriptive what implementation does and we do not infect it directly, all interactions are through `DocumentsService`. You still wanna rename it? We could use something like `DefaultDocumentsService` (as @dharmendrak) suggested if you feel strong about renaming.

Reply:
I'm fine with keeping it, none of those names feels 100% alright 😉

Reply:
OntheFly is also called inflight which was a good suggestion I think. Nevertheless, Damian's name is also fine being slightly more descriptive.